docs/dev-notes/DEVELOPMENT.md

```diff
@@ -21,6 +21,13 @@ To get started with the project, you might need several tools to compile on your
    - Load rbenv in your shell by running `rbenv init` and restarting the shell
    - Make sure you have the required ruby version set locally. You can do this by running `rbenv local 2.7.6`
  - Make sure you have [Xcode Command Line Tools installed](https://stackoverflow.com/a/45566089)
+ - Make sure you have JDK installed on your machine. Recommended JDK versions is 11, if you encounter issues with Java/Kotlin files compilation, please upgrade/downgrade JDK version to the recommended one.
```

Comment:
Can we have a link to an external resource where the recommended JDK version for React Native is? 

That way we can keep this file up-to-date with the currently recommended JDK version?

Reply:
[Added link](https://github.com/customerio/amiapp-reactnative/pull/52/commits/82b56df0d032a30d6dd4a2f6bd2a7548cfa83d58) to react native docs.